cpu/esp_common/Makefile.dep

```diff
@@ -72,6 +72,10 @@ ifneq (,$(filter esp_freertos,$(USEMODULE)))
   USEMODULE += esp_freertos_common
 endif
 
+ifneq (,$(filter esp_freertos_common,$(USEMODULE)))
+  USEMODULE += ztimer_msec
+endif
+
```

Comment:
Please remove it, it shouldn't be needed here because it wasn't needed with `xtimer`.

Reply:
Ah no guarded against `MODULE_ESP_WIFI_ANY` ok